rdr_service/dao/genomics_dao.py

```diff
@@ -3245,3 +3256,120 @@ def get_w3sr_records(self, **kwargs):
+            query = session.query(
+                func.concat(get_biobank_id_prefix(), GenomicSetMember.biobankId),
+                GenomicSetMember.sampleId,
+                GenomicGCValidationMetrics.hfVcfPath.label('vcf_raw_path'),
+                GenomicGCValidationMetrics.hfVcfTbiPath.label('vcf_raw_index_path'),
+                GenomicGCValidationMetrics.hfVcfMd5Path.label('vcf_raw_md5_path'),
+                GenomicGCValidationMetrics.cramPath.label('cram_name'),
+                func.IF(
+                    GenomicSetMember.nyFlag == 1,
+                    sqlalchemy.sql.expression.literal("Y"),
+                    sqlalchemy.sql.expression.literal("N")
+                ).label('ny_flag'),
+                sqlalchemy.func.upper(GenomicSetMember.gcSiteId).label('genome_center'),
+                sqlalchemy.case(
+                    [(ParticipantSummary.consentForGenomicsROR == QuestionnaireStatus.SUBMITTED, 'Y')],
+                    else_='N'
+                ).label('consent_for_gror'),
+                sqlalchemy.literal('aou_cvl').label('genome_type'),
+                sqlalchemy.case(
+                    [(informing_loop_subquery.decision_value.like('yes'), 'Y')],
+                    else_='N'
+                ).label(f'informing_loop_{module}'),
+                GenomicGCValidationMetrics.aouHdrCoverage.label('aou_hdr_coverage'),
+                GenomicGCValidationMetrics.contamination
+            ).join(
+                ParticipantSummary,
+                ParticipantSummary.participantId == GenomicSetMember.participantId
+            ).join(
+                GenomicGCValidationMetrics,
+                and_(
+                    GenomicGCValidationMetrics.genomicSetMemberId == GenomicSetMember.id,
+                    GenomicGCValidationMetrics.ignoreFlag != 1
+                )
+            ).join(
+                BiobankStoredSample,
+                BiobankStoredSample.biobankStoredSampleId == GenomicSetMember.collectionTubeId
+            ).join(
+                BiobankOrderIdentifier,
+                BiobankOrderIdentifier.value == BiobankStoredSample.biobankOrderIdentifier
+            ).join(
+                BiobankOrder,
+                BiobankOrder.biobankOrderId == BiobankOrderIdentifier.biobankOrderId
+            ).join(
+                sample_collected_site,
+                sample_collected_site.siteId == BiobankOrder.collectedSiteId
+            ).join(
+                informing_loop_subquery,
+                informing_loop_subquery.participant_id == GenomicSetMember.participantId
+            ).outerjoin(
+                GenomicResultWorkflowState,
+                and_(
+                    GenomicResultWorkflowState.genomic_set_member_id == GenomicSetMember.id,
+                    GenomicResultWorkflowState.results_module == result_module_type
+                )
+            ).filter(
+                GenomicGCValidationMetrics.processingStatus.like('pass'),
```

Comment:
For this and the other filters using `like`, we don't transform the values to `lower` when we ingest, so these filters will need to be case insensitive. Though `LIKE` is case insensitive in MySql, if I'm not mistaken, SqlAlchemy's `like()` is case sensitive. You'll probably want to use `ilike()` here, which is insensitive, or wrap it in a func.lower() (which we do elsewhere): `func.lower(GenomicGCValidationMetrics.processingStatus) == "pass",`. 
This is also true for `sexConcordance` ("TRUE"), `drcSexConcordance` ("PASS"), and `gcManifestSampleSource` ("Whole Blood").

Reply:
Yea, neither did I. Apparently PosgreSQL's `LIKE` is case sensitive, and so it has the operator `ILIKE`. For systems that don't have `ILIKE`, like MySQL, SqlAlchemy wraps it in a `lower()` under the hood. The reason we don't mutate the values to lower is because if we have to pass these through, we have to follow the manifest specs, as un-standardized as they may be :)